client/app.ts

```diff
@@ -0,0 +1,45 @@
+import 'reflect-metadata';
+import 'zone.js/dist/zone';
+import { Component, provide } from '@angular/core';
+import { bootstrap } from 'angular2-meteor-auto-bootstrap';
+import { ROUTER_PROVIDERS, ROUTER_DIRECTIVES, RouteConfig, RouterLink } from '@angular/router-deprecated'; 
+import { APP_BASE_HREF } from '@angular/common';
+import { Account } from './imports/account/account.ts';
+import { Assignment } from './imports/assignment/assignment.ts';
+import { Grade } from './imports/grade/grade.ts';
+import { Login } from './imports/login/login.ts';
+import { Home } from './imports/home/home.ts';
+import { CoursePage } from './imports/coursepage/coursepage.ts';
+import {MATERIAL_PROVIDERS, MATERIAL_DIRECTIVES} from 'ng2-material';
+import {MdToolbar} from '@angular2-material/toolbar';
+import '../node_modules/@angular2-material/button/button.css';
+import '../node_modules/@angular2-material/toolbar/toolbar.css';
+
+ @Component({
+   selector: 'app',
+   templateUrl: 'client/app.html',
+   directives: [ROUTER_DIRECTIVES, MATERIAL_DIRECTIVES, MdToolbar]
+ })
+
+ // @Component({
+ //   selector: 'app',
+ //   templateUrl: 'client/app.html',
+ //   directives: [ROUTER_DIRECTIVES, MATERIAL_DIRECTIVES, MdToolbar]
+ // })
+
+@RouteConfig([
+  { path: '/login', component: Login},
+  { path: '/account', component: Account},
+  { path: '/assignment', component: Assignment},
+  { path: '/grade', component: Grade},
+  { path: '/home', component: Home},
+  { path: '/coursepage', component: CoursePage}
```

Comment:
This does not meet the specification you were given to generate dynamic routes (which is covered in the tutorial http://www.angular-meteor.com/tutorials/socially/angular2/routing-and-multiple-views#configuringroutes).  How else are we to know which course is being looked at?